### PR TITLE
Upgrade to Java 11

### PR DESCRIPTION
**Pull request by jkosternl**

### CODE REVIEW

**Comment by nielsm5** [comment]
Please also remove the docker websphere related things :)

**Comment by nielsm5** [comment]
And lcm the other docker images

**Comment by tnleeuw** [comment]
> Please also remove the docker websphere related things :)

There is also various bits of code to support WAS spread around the system. Perhaps it's better to remove all of that in a separate PR?
(Just a suggestion).

File: .github/dependabot.yml
Comment (tnleeuw) [inline .github/dependabot.yml:1]: The lock on dependencies that is now for target-branch 7.8-release also needs to be on 7.9-release branch which will be made soon.

Comment (jkosternl) [reply .github/dependabot.yml:1]: Yes indeed. But that's for another PR 😉 likely for somebody else.

Comment (tnleeuw) [reply .github/dependabot.yml:1]: We need to keep this in the back of our minds though.

**Comment by philipsens** [comment]
I marked the PR as ready to review to test the pipelines

File: docker/JBoss/docker-compose.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ services:
         image: iaf-test-as-jboss
         restart: unless-stopped
         ports:
-            - "80:8080"
\ No newline at end of file
+            - "8080:8080"
+            - "9990:9990"
```
[the remark this answers is not in the document]
Comment (jkosternl) [reply docker/JBoss/docker-compose.yml]: Done.

**Review by nielsm5 — APPROVED** [review]

**Comment by nielsm5** [comment]
I've removed some changes and will push them to another branch, to be solved at a later stage.

**Comment by jkosternl** [comment]
> I've removed some changes and will push them to another branch, to be solved at a later stage/in a separate branch.

Totally fine! The Dockerfile with JBoss 7.4 was also not finished yet.

**Comment by tnleeuw** [comment]
The JBoss build was aborted, not sure what's wrong with that but it looks like the application didn't start up?

https://jenkins.frankframework.org/job/ff-ci/job/matrix-pr/653/APPSERVER=jboss,DBMS=mysql,MQ=activemq/console

image


https://jenkins.frankframework.org/job/ff-ci/job/larva/15454/console

```
12:05:10  [Pipeline] {
12:05:10  [Pipeline] script
12:05:10  [Pipeline] {
12:05:10  [Pipeline] httpRequest
12:05:10  HttpMethod: GET
12:05:10  URL: http://localhost:8080/iaf-test/iaf/api/server/health
12:05:10  Sending request to url: http://localhost:8080/iaf-test/iaf/api/server/health
12:05:10  Response Code: HTTP/1.1 404 Not Found
12:05:10  Success: Status code 404 is in the accepted range: 100:599
```

I guess we need to investigate that before this PR can be merged?

**Comment by nielsm5** [comment]
> The JBoss build was aborted, not sure what's wrong with that but it looks like the application didn't start up?
> 
> I guess we need to investigate that before this PR can be merged?
No, we should merge this first and solve the JBoss issues in another PR.

**Comment by tnleeuw** [comment]
> > The JBoss build was aborted, not sure what's wrong with that but it looks like the application didn't start up?
> > I guess we need to investigate that before this PR can be merged?
> 
> No, we should merge this first and solve the JBoss issues in another PR.

Even the issue there is now after you reverted the JBoss 7.4 changes?
Looks like this will most likely break the build on master then if you merge it as-is?

**Review by tnleeuw — APPROVED** [review]

**Comment by nielsm5** [comment]
Yup, #5654

**Pull request merged**